src/main/java/uk/gov/justice/hmpps/prison/service/AgencyServiceImpl.java

```diff
@@ -281,9 +291,9 @@ public AgencyEstablishmentTypes getEstablishmentTypes(final String agencyId) {
                 .build();
     }
 
-    private OffenderCell transform(LivingUnit livingUnit) {
+    private OffenderCell transform(LivingUnit livingUnit, Boolean checkCapacity) {
         final var agencyLocation = agencyInternalLocationRepository.findOneByDescription(livingUnit.getDescription());
-        if (agencyLocation.isPresent() && agencyLocation.get().isActiveCellWithSpace()) {
+        if (agencyLocation.isPresent() && (!checkCapacity || agencyLocation.get().isActiveCellWithSpace())) {
```

Comment:
might be better to use a primitive if you want `checkCapacity` to never be null

Reply:
Thank you, I've done that :)